.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features
+          args: --all-features --all-targets
```

Comment:
Unless I'm missing something, it looks like `--all-targets` causes doc tests to not run.  Ran into this with another project. when comparing `cargo test` (fails) to `cargo test --all-targets` (passes).

:(

Reply:
yeah: https://github.com/rust-lang/cargo/issues/6669

Reply:
Looks like we are covered with our default feature run, so probably good enough here.